p2p/src/peers/mod.rs

```diff
@@ -256,6 +256,11 @@ impl Peers {
         addrs
             .iter()
             .filter_map(|address| {
+                // Add to iced
+                if ice {
+                    self.ice_peer_address(address);
+                }
```

Comment:
Can we add a test for this change? Check that when trying to remove an address that did not exist, this address will be iced.

Reply:
sure